Makefile

```diff
@@ -613,24 +593,24 @@ check-packr: bpf/packrd/packed-packr.go
 		false; \
 	fi
 
+SOURCE_DIRS := "./bpf/ ./buildinfo/ ./calc/ ./check-licenses/ ./cmd/ ./config/ ./conntrack/ ./daemon/ ./dataplane/ ./dispatcher/ ./fv/ ./hashutils/ ./ifacemonitor/ ./ip/ ./ipsets/ ./iptables/ ./jitter/ ./k8sfv/ ./labelindex/ ./logutils/ ./markbits/ ./multidict/ ./policysync/ ./proto/ ./routetable/ ./rules/ ./statusrep/ ./stringutils/ ./testutils/ ./throttle/ ./usagerep/ ./versionparse/"
+
 # Run go fmt on all our go files.
 .PHONY: go-fmt goimports fix
 fix go-fmt goimports:
-	$(DOCKER_RUN) $(CALICO_BUILD) sh -c 'glide nv -x | \
-	      grep -v -e "^\\.$$" | \
-	      xargs goimports -w -local github.com/projectcalico/'
+	$(DOCKER_RUN) $(CALICO_BUILD) sh -c 'echo $(SOURCE_DIRS) | xargs goimports -w -local github.com/projectcalico/'
 
 .PHONY: check-typha-pins
 check-typha-pins: vendor/.up-to-date
 	@echo "Checking Typha's libcalico-go pin matches ours (so that any datamodel"
 	@echo "changes are reflected in the Typha-Felix API)."
 	@echo
 	@echo "Felix's libcalico-go pin:"
-	@grep libcalico-go glide.lock -A 5 | grep 'version:' | head -n 1
+	@grep libcalico-go go.mod | awk '{print $$2}'
 	@echo "Typha's libcalico-go pin:"
-	@grep libcalico-go vendor/github.com/projectcalico/typha/glide.lock -A 5 | grep 'version:' | head -n 1
-	if [ "`grep libcalico-go glide.lock -A 5 | grep 'version:' | head -n 1`" != \
-	     "`grep libcalico-go vendor/github.com/projectcalico/typha/glide.lock -A 5 | grep 'version:' | head -n 1`" ]; then \
+	@grep libcalico-go vendor/github.com/projectcalico/typha/go.mod | awk '{print $$2}'
+	if [ "`grep libcalico-go vendor/github.com/projectcalico/typha/go.mod | awk '{print $$2}'`" != \
```

Comment:
You can access the downloaded contents of dependencies using something like

```
cat $(go list -m -f "{{.Dir}}" github.com/projectcalico/typha)/glide.lock
```

This makes it so we don't need `vendor` for this kind of thing

Or, specifically for the go.mod file

```
cat $(go list -m -f "{{.GoMod}}" github.com/projectcalico/typha)
```
```

Reply:
Fixed